setup.py

```diff
@@ -6,12 +6,15 @@
 
 description = "The official Nomic python client."
 
+with open('README.md') as f:
+    long_description = f.read()
+
 setup(
     name="nomic",
-    version="3.3.0",
```

Comment:
You don't need to bump the version, release please will handle it

Reply:
Oh yeah +1